src/common/common/display.h

```diff
@@ -228,5 +235,27 @@ void guac_common_display_free_layer(guac_common_display* display,
 void guac_common_display_free_buffer(guac_common_display* display,
         guac_common_display_layer* display_buffer);
 
+/**
+ * Sets the overall lossless compression policy of the given display to the
+ * given value, affecting all current and future layers/buffers maintained by
+ * the display. By default, newly-created displays will use lossy compression
+ * for graphical updates when heuristics determine that doing so is
+ * appropriate. Specifying a non-zero value here will force all graphical
+ * updates to always use lossless compression, whereas specifying zero will
+ * restore the default policy.
+ *
+ * Note that this can also be adjusted on a per-layer / per-buffer basis with
+ * guac_common_surface_set_lossless().
+ *
+ * @param display
+ *     The display to modify
```

Comment:
Looks like there's a period missing here on this param doc - same as `guac_common_surface_set_lossless`.

Reply:
Oops. Fixed and rebased.